pages/store/index.vue

```diff
@@ -567,7 +594,11 @@ async function fetchItems({ lazy = false, isRefresh = false } = {}) {
   }
 
   try {
-    await bookstoreStore.fetchCMSProductsByTagId(localizedTagId.value, { isRefresh })
+    await Promise.all([
+      bookstoreStore.fetchCMSProductsByTagId(localizedTagId.value, { isRefresh }),
+      // NOTE: Fetch staking books for sorting items
+      bookstoreStore.fetchStakingBooks('total_staked', { isRefresh }).catch(() => {}),
```

Comment:
`fetchStakingBooks` is a paginated API so there is not guarantee it would include all books in a tag, maybe we also need some kind of lazy fetching?
For now maybe we should at least set `fetchStakingBooks` page size to max (100)